osgp/platform/osgp-secret-management/src/main/java/org/opensmartgridplatform/secretmanagement/application/exception/DetailSoapFaultMappingExceptionResolver.java

```diff
@@ -44,4 +48,18 @@ protected void customizeFault(final Object endpoint, final Exception ex, final S
         }
     }
 
+    @Override
+    protected void logException(final Exception ex, final MessageContext messageContext) {
+        log.error("[{}] Exception occurred during SOAP request processing", this.getCorrelationUid(messageContext), ex);
+    }
+
+    private String getCorrelationUid(final MessageContext messageContext) {
+        final SaajSoapMessage request = (SaajSoapMessage) messageContext.getRequest();
+        final Iterator<SoapHeaderElement> iter = request.getEnvelope()
+                .getHeader()
+                .examineHeaderElements(new QName("correlationUid"));
```

Comment:
This doesn't work, should be `new QName("http://www.opensmartgridplatform.org/schemas/security/secretmanagement", "correlationUid")` (including the namespace) to match the correlation UID header element.

Reply:
new QName(NAMESPACE_URI, CORRELATION_UID)
NAMESPACE_URI, CORRELATION_UID are now static imports from SecretManagementEndpoint class